lib/dequeue.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+module.exports = function (options) {
+  var seneca = this
+
+  seneca.add('role:updater,info:update', onNpmChange)
+
+  function onNpmChange (pkg, respond) {
```

Comment:
Move function outside plugin, name of function should just be dequeue or infoUpdated.